fedora/Dockerfile.amd64

```diff
@@ -1,10 +1,53 @@
 ARG BASE=fedora
 ARG BASETAG=39
 
+FROM ${BASE}:${BASETAG} AS stage-build
+
 ARG ARG_CLIENT_BRANCH
 ARG ARG_CLIENT_BUILD_TARGET
 
-FROM ${BASE}:${BASETAG} as stage-fedora
+ENV OWNBUILD_DIR=/ownbuild
+ENV CLIENT_BRANCH=${ARG_CLIENT_BRANCH:-master}
+ENV CLIENT_BUILD_TARGET=${ARG_CLIENT_BUILD_TARGET:-linux-gcc-x86_64-squish}
+
+RUN dnf install -y \
+    git-core \
+    cmake \
+    ninja-build \
+    libEGL-devel \
+    gcc \
+    gcc-c++ \
+    libxkbcommon-devel \
+    libxkbcommon-x11-devel \
+    xcb-util*
+
+RUN curl https://dl.fedoraproject.org/pub/fedora/linux/releases/39/Everything/x86_64/os/Packages/p/python3.10-3.10.13-1.fc39.x86_64.rpm \
+    -so python3.10.13.rpm && \
+    dnf install -y ./python3.10.13.rpm
+
+RUN mkdir -p ${OWNBUILD_DIR} && \
+    curl https://raw.githubusercontent.com/owncloud/ownbuild/master/ownbuild.py -so ${OWNBUILD_DIR}/ownbuild.py && \
+    python3.10 ${OWNBUILD_DIR}/ownbuild.py \
+    --branch ${CLIENT_BRANCH} \
+    --target ${CLIENT_BUILD_TARGET} \
+    -- \
+    --options "libs/llvm.ignored=True" \
+    --install-deps owncloud-client && \
+    # remove unnecessary files
+    cd ${OWNBUILD_DIR}/${CLIENT_BRANCH}/${CLIENT_BUILD_TARGET} && \
+    mv ./lib/gettext/hostname ./dev-utils/bin/hostname && \
+    rm -rf build home logs craft translations && \
+    rm -rf ./lib/libpython* ./lib/gettext ./lib/perl* ./lib/python3* && \
```

Comment:
It might also work to skip them directly like llvm.
`--options "libs/python.ignored=True"`
`--options "dev-utils/python3.ignored=True"`
`--options "dev-utils/perl.ignored=True"`